x/ibc/applications/transfer/types/codec.go

```diff
@@ -4,12 +4,15 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
 // RegisterInterfaces register the ibc transfer module interfaces to protobuf
 // Any.
 func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil), &MsgTransfer{})
+
+	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
```

Comment:
We should make sure this addition is accounted for in the "building a module" docs that @blushi is updating in #7660